core/src/main/java/be/isach/ultracosmetics/cosmetics/particleeffects/ParticleEffectMagicalRods.java

```diff
@@ -47,6 +47,8 @@ public class ParticleEffectMagicalRods extends ParticleEffect {
 
     public ParticleEffectMagicalRods(UltraPlayer owner, ParticleEffectType type, UltraCosmetics ultraCosmetics) {
         super(owner, type, ultraCosmetics);
+
+        this.displayIfMoving = false;

```

Comment:
Why doesn't this display at all while moving (not even a small effect like the others?)

Reply:
I disabled the alternative effect because the trail was buggy, it had a red particle trail like the BloodHelix.
I fixed the MagicalRods trail. It even has the same colors as the default effect 😃.

![fixedmagicalrodstrail](https://user-images.githubusercontent.com/33426785/210160310-508363d5-7274-4e0f-b6b4-e0670601d818.gif)